azext_iot/operations/hub.py

```diff
@@ -617,6 +622,28 @@ def _process_config_content(content, content_type='module'):
     return content
 
 
+def _validate_payload_schema(content):
+    from jsonschema import validate
+    from jsonschema.exceptions import ValidationError, SchemaError
+    from azext_iot._constants import EDGE_DEPLOYMENT_SCHEMA_2_PATH as schema_path
+
+    try:
+        logger.info("Validating deployment payload...")
```

Comment:
NIT: I may move this logger after the following if block. Seems to make more sense to me to see this kind of in progress log only if it actually validating.

Reply:
Moved.